tensorflow/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -3674,20 +3675,33 @@ GetCudnnFusedOperationGraph(
                      .build();
   RETURN_MSG_IF_CUDNN_ERROR(conv_op);
 
-  auto add_desc = cudnn_frontend::PointWiseDescBuilder()
-                      .setMode(CUDNN_POINTWISE_ADD)
-                      .setMathPrecision(cudnn_activation_type)
-                      .build();
-  auto add_op = cudnn_frontend::OperationBuilder(
-                    CUDNN_BACKEND_OPERATION_POINTWISE_DESCRIPTOR)
-                    .setxDesc(conv_op.getOutputTensor())
-                    .setbDesc(tensor_z)
-                    .setyDesc(tensor_add)
-                    .setpwDesc(add_desc)
-                    .setAlpha(alpha)
-                    .setAlpha2(alpha2)
-                    .build();
-  RETURN_MSG_IF_CUDNN_ERROR(add_op);
+  // CUDNN OperationGraph
+  absl::InlinedVector<cudnn_frontend::Operation const*, 4> ops = {&conv_op};
+
+  bool skip_add_op = activation_mode == dnn::ActivationMode::kRelu6 ||
+                     activation_mode == dnn::ActivationMode::kElu ||
+                     activation_mode == dnn::ActivationMode::kLeakyRelu;
```

Comment:
Invert this logic?  We seem to be saying that the add op is compatible only with certain activations, not that the add op is *incompatible* with this list of activations but will work with any other ones...

(This isn't academic: I notice that we have not implemented the kBandPass activation, but based on this CL that seems straightforward and something we should do.  I presume that kBandPass should act just like the activations listed here WRT skip_add_op.)